naming/src/main/java/com/alibaba/nacos/naming/controllers/InstanceController.java

```diff
@@ -433,6 +440,19 @@ public ObjectNode listWithHealthStatus(@RequestParam String key) throws NacosExc
         return result;
     }
     
+    private void checkServiceNameFormat(String serviceName) {
+        try {
+            String[] split = serviceName.split(Constants.SERVICE_INFO_SPLITER);
+            String service = split[1];
```

Comment:
I think check length of array is better than catch exception.

Reply:
agree